src/platform/packages/shared/kbn-ftr-common-functional-services/services/saml_auth/saml_auth_provider.ts

```diff
@@ -126,6 +126,12 @@ export function SamlAuthProvider({ getService }: FtrProviderContext) {
       return sessionManager.getApiCredentialsForRole(role, options);
     },
 
+    async getM2MApiCookieCredentialsWitCustomRoleScope(
```

Comment:
We're likely missing an `h` in `With` here:

```suggestion
    async getM2MApiCookieCredentialsWithCustomRoleScope(
```